addons/html_builder/static/src/sidebar/block_tab.js

```diff
@@ -203,11 +203,18 @@ export class BlockTab extends Component {
         const iframeWindow =
             this.document.defaultView !== window ? this.document.defaultView : false;
 
-        const scrollingElement = () =>
-            this.shared.dropzone.getDropRootElement() ||
-            this.editable.querySelector(".o_notebook") ||
-            getScrollingElement(this.document) ||
-            this.editable.querySelector(".o_editable");
+        const scrollingElement = () => {
```

Comment:
Question: why is it the whole page scolling instead of the iframe in mass_mailing ? I would find it more logical to have the scroll bar here (like for the sidebar) instead no ? and it would be more compatible with the builder since it would be inside the iframe too, no ? 
<img width="1047" height="898" alt="image" src="https://github.com/user-attachments/assets/70ad0460-cad0-4864-b5e5-f27f5c422e34" />

Like keep the "form view" above visible and fixed (it is not that big so it won't be annoying) and make only the iframe/notebook scroll ? :thinking: it would simplify everything and prevent almost all the changes of this commit :thinking: